front/migrations/db/migration_72.sql

```diff
@@ -0,0 +1,5 @@
+-- Migration created on Sep 03, 2024
+ALTER TABLE agent_tables_query_configuration_tables
```

Comment:
you didn't set the migration as mandatory (via create-migration cli) ?

Reply:
you'll find a template you just have to copy paste and put your migration in the middle.

Reply:
Yep updated.

Reply:
❤️